### PR TITLE
[ES] Nimble publisher Spanish translation

### PR DESCRIPTION
**Pull request by jaimeiniesta**
I've made minor corrections in the NimblePublisher lesson, and translated it to Spanish.

### CODE REVIEW

File: es/lessons/libraries/nimble-publisher.md [hunk 1]
```diff
@@ -0,0 +1,358 @@
...
+
+NimblePublisher también puede ser empleado para construir otros contextos de publicación con una estructura diferente.
+
+Por ejemplo, podríamos mantener una colección de Preguntas Frecuentes (FAQs), en este caso probablemente no necesitemos fechas ni autores, y una estructura con `:id`, `:question` and `:answer` sería suficiente:
+
+Podríamos colocar nuestros ficheros de contenidos en una estructura de directorio diferente, por ejemplo:
+
+```
+/priv/faqs/is-there-a-free-trial.md
+/pric/faqs/when-did-it-start.md
```
Comment (erickgnavar) [inline es/lessons/libraries/nimble-publisher.md:319]:
```suggestion
/priv/faqs/when-did-it-start.md
```

File: en/lessons/libraries/nimble-publisher.md [hunk 2]
```diff
@@ -318,7 +318,7 @@ We could place our content files on a different directory structure, for example
 /pric/faqs/when-did-it-start.md
```
Comment (erickgnavar) [inline en/lessons/libraries/nimble-publisher.md:318]:
```suggestion
/priv/faqs/when-did-it-start.md
```

**Review by erickgnavar — DISMISSED** [review]
LGTM 👍

**Comment by jaimeiniesta** [comment]
Good catches @erickgnavar thanks!

**Review by erickgnavar — APPROVED** [review]

**Pull request merged**